packages/material-ui-lab/src/Autocomplete/Autocomplete.js

```diff
@@ -28,13 +28,19 @@ export const styles = (theme) => ({
   focused: {},
   /* Styles applied to the tag elements, e.g. the chips. */
   tag: {
-    margin: 3,
+    margin: '4px 4px 4px 0',
```

Comment:
I'm confused, how is the padding related to the font issue we were discussing?

Reply:
It was required to get it unified with increased Input height. Without a jumping was present when there was input on the not-first line like here
![image](https://user-images.githubusercontent.com/6066417/81101831-5ae1f900-8f17-11ea-8374-1cfc04c5e511.png)
Horizontaly it was added for unified view. BTW it is now more aligned with the 8px modular grid
I can try to evade it

Reply:
What's the origin of the increase of the input height?

Reply:
Well the input line-height was some value from UA and it was different for `<input />` `<textarea />`, their placeholders and  `<select />`, native and non native. I noticed that it causes out-of-grid 1-2px irregularities and much more for `multiline` so I unified them by doing this https://github.com/mui-org/material-ui/pull/20908/commits/347503800004ed61a9eb04bce11c62337c786c70
Specifically for Autocomplete I can revert them back to something like `19.2px` value if it is necessary